### PR TITLE
novacustom: fan profiles description

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

**Review by BeataZdunczyk — APPROVED** [review]

File: docs/unified/novacustom/fan-profiles.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+| 100              | 100           |
+
+## Performance fan profile
+
+| Temperature [°C] | Fan speed [%] |
+|------------------|---------------|
+| 0                | 25            |
+| 55               | 35            |
+| 75               | 60            |
+| 100              | 100           |
```
Comment (sulewskiprzemyslaw) [inline docs/unified/novacustom/fan-profiles.md:23]: @macpijan What is the characteristic between the indicated points? Linear? I think such information would be helpful for users.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified/novacustom/fan-profiles.md (hunk 1 above)
Comment (mkopec) [reply docs/unified/novacustom/fan-profiles.md:23]: Values inbetween curve points are linearly interpolated, yes.

For temperatures below the first point the fans will be stoppend and above the last point the fans will be set to 100%.

Comment (sulewskiprzemyslaw) [reply docs/unified/novacustom/fan-profiles.md:23]: So we should add such an information

Comment (mkopec) [reply docs/unified/novacustom/fan-profiles.md:25]: Added a note about this.

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**